app/models/user.rb

```diff
@@ -28,6 +28,8 @@ class User < ApplicationRecord
         -> { joins(:notification_timings).merge(NotificationTiming.attended_to_event) }
   scope :allowing_liked_event_notification,
         -> { joins(:notification_timings).merge(NotificationTiming.liked_event) }
+  
+  enum gender: {other: 0, woman: 1, man: 2}
```

Comment:
```suggestion
  enum gender: { other: 0, woman: 1, man: 2 }
```
Rubyでは { } の最初と最後に空白が入っているのが通例なので（動作上問題ないのですが）